Data/models/Domain.swift

```diff
@@ -190,10 +190,6 @@ extension Domain {
                     context.delete($0)
                 }
             }
-            for obj in results {
-                // Cascading delete on favicon, it will also get deleted
-                context.delete(obj)
```

Comment:
@kylehickinson - I'm not able to trigger the code on master. But, not deleting the `obj` would also retain the other attributes of the object like url, visits and historyItems in addition to favicon. Let me know if I'm missing something here?

Reply:
@jumde the object is still deleted if it not a Domain that belongs to a Bookmark/fav (as seen in logic above)

For Domain's that do it just resets the visit count which was the intention of this function (but has just been bugged for a year)

Reply:
Makes sense. This code confused me. I was thinking why other attributes of `Domain` are not being reset.

```
if let bms = $0.bookmarks, bms.count > 0 {
                    // Clear visit count
                    $0.visits = 0
                }
```